payments-core/src/main/java/com/stripe/android/IssuingCardPinService.kt

```diff
@@ -19,6 +19,7 @@ import kotlin.coroutines.CoroutineContext
 /**
  * Methods for retrieval / update of a Stripe Issuing card
  */
+@Deprecated("Please use Issuing Elements instead: https://stripe.com/docs/issuing/elements")
```

Comment:
Should we link to https://stripe.com/docs/issuing/elements#issuing-elements-and-native-applications instead?

Reply:
Though this section is immediately relevant to native apps, I think it's better to link to the top of that same doc, as its likely the user won't even know what Issuing Elements is or how it's related to PINs.

Additionally, we've been discussing the possibility of building native Issuing Elements, and if/when that happens, it'll be documented at https://stripe.com/docs/issuing/elements and https://stripe.com/docs/issuing/elements#issuing-elements-and-native-applications will likely be removed